sdk/search/search/test/base64.spec.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert } from "chai";
+import { encode, decode } from "../src/base64";
+
+describe("base64", () => {
+  it("strings can roundtrip", () => {
+    const message = "Only *you* can prevent null dereferences!";
```

Comment:
I'm pretty sure [this is Tony Hoare's fault](https://en.wikipedia.org/wiki/Tony_Hoare#Apologies_and_retractions) and not mine...